backend/app/models/question.py

```
@@ -1,11 +1,91 @@
-from sqlmodel import Field, SQLModel
+from sqlmodel import Field, SQLModel, Relationship
+from typing import Optional, TYPE_CHECKING
+from sqlalchemy import Column, JSON, DateTime
+from datetime import datetime, timezone
+from sqlalchemy import Column
```

Ruff (0.12.2)

Redefinition of unused `Column` from line 3

Remove definition: `Column`

(F811)